### PR TITLE
Update Dockerfile-debian

### PR DESCRIPTION
**Pull request by WinterSilence**
Remove default/installed PHP extensions, see https://github.com/mlocati/docker-php-extension-installer#supported-php-extensions
Add PHP extensions: `sockets` and `gettext`
Install Composer via `mlocati/php-extension-installer`

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: php/Dockerfile-debian [hunk 1]
```diff
@@ -4,28 +4,31 @@
...
+        gd \
+        imagick \
+        opcache \
         xdebug
 
 # Disable xdebug by default (see PHP_ENABLE_XDEBUG)
 RUN rm /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
 
+# Install Composer
+RUN install-php-extensions @composer-2.1.3
```
Comment (schmunk42) [inline php/Dockerfile-debian]: @WinterSilence 
Do we need to specify a version here?

Comment (WinterSilence) [reply php/Dockerfile-debian]: @WinterSilence my fail - i use fixed versions and missed update this part when copy code

**Comment by schmunk42** [comment]
@WinterSilence 

> Add PHP extensions: sockets and gettext

Are those required or recommended, which code in Yii uses them?

**Comment by WinterSilence** [comment]
@schmunk42 gettext - translator-message-gettext , sockets - yii-queue-amqp

**Comment by schmunk42** [comment]
@WinterSilence Please see **Checks**, they are failing.

```
Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "docker-php-entrypoint": executable file not found in $PATH: unknown
```

**Comment by WinterSilence** [comment]
@schmunk42  fixed

**Review by schmunk42 — APPROVED** [review]

**Pull request closed without merging**